tests/lib/rules/no-invalid-regexp.js

```diff
@@ -26,11 +26,11 @@ ruleTester.run("no-invalid-regexp", rule, {
         { code: "new RegExp('.', 'y')", options: [{ allowConstructorFlags: ["y"] }]},
         { code: "new RegExp('.', 'u')", options: [{ allowConstructorFlags: ["U"] }]},
         { code: "new RegExp('.', 'yu')", options: [{ allowConstructorFlags: ["y", "u"] }]},
-        { code: "new RegExp('\/', 'yu')", options: [{ allowConstructorFlags: ["y", "u"] }]},
+        { code: "new RegExp('\/', 'yu')", options: [{ allowConstructorFlags: ["y", "u"] }]}, // eslint-disable-line no-useless-escape
```

Comment:
I honestly cannot tell how this line is different from the one above it. Here's the original commit: https://github.com/eslint/eslint/commit/c0a7be609e420c6939330909bdb43d49b3670cf5#diff-9f7fae656233bad860f2ba8d46986835R26
Those tests were added to test the new `y` and `u` flags, as it seems.